app/services/doc_auth/classification_concern.rb

```diff
@@ -0,0 +1,36 @@
+module DocAuth
+  module ClassificationConcern
+    extend ActiveSupport::Concern
+    # check whether doc type is supported, return false only if classification info clearly
+    # identifies the document type and that document type is not a supported one
+    def id_type_supported?
+      info = classification_info
+      return true if info.nil?
+      type_ok = doc_side_class_ok?(info, 'Front') && doc_side_class_ok?(info, 'Back')
+      issuing_country_ok = doc_issuing_country_ok?(
+        info,
+        'Front',
+      ) && doc_issuing_country_ok?(info, 'Back')
+      type_ok && issuing_country_ok
+    end
+
+    alias_method :doc_type_supported?, :id_type_supported?
```

Comment:
I think I'd prefer to use the same method name rather than aliasing?

Reply:
@charleyf , the alias method used in other places, thinking clean it up with following ticket to avoid bigger impact.